File: ITapply/ITapply.Models/SearchObjects/EmployerSkillSearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ITapply.Models.SearchObjects
{
    public class EmployerSkillSearchObject : BaseSearchObject
    {
        public int? EmployerId { get; set; }
        public int? SkillId { get; set; }
    }
} 

File: ITapply/ITapply.UI/itapply_mobile/lib/models/search_objects/employer_skill_search_object.dart
import 'package:itapply_desktop/models/search_objects/base_search_object.dart';
import 'package:json_annotation/json_annotation.dart';

part 'employer_skill_search_object.g.dart';

@JsonSerializable()
class EmployerSkillSearchObject extends BaseSearchObject {
  int? EmployerId;
  int? SkillId;

  EmployerSkillSearchObject({
    this.EmployerId,
    this.SkillId,
    super.Page,
    super.PageSize,
    super.IncludeTotalCount,
    super.RetrieveAll,
  });

  @override
  Map<String, dynamic> toJson() => _$EmployerSkillSearchObjectToJson(this);
}